src/main.rs

```diff
@@ -14,6 +14,7 @@ mod install;
 mod installer;
 mod settings;
 mod terminal;
+mod workers;
```

Comment:
this should be singular in my opinion

Reply:
It was at one point and I think @ashleymichal wanted it to be plural, but I can't remember why. Maybe she could chime in?

Reply:
the idea was to distinguish based on the platform (Workers is the name of the platform/product) in direct contrast to, say, KV, which is another platform Wrangler will interact with. @ashleygwilliams and I discussed this morning and there is a suggested structure of project that may help.